pages/[channel]/[locale]/category/[slug].tsx

```diff
@@ -62,22 +42,32 @@ const CategoryPage = ({
 
 export default CategoryPage;
 
-export async function getStaticPaths() {
-  const result: ApolloQueryResult<CategoryPathsQuery | undefined> =
-    await apolloClient.query({
-      query: CategoryPathsDocument,
-      variables: {},
-    });
-  const paths =
-    result.data?.categories?.edges.map(({ node }) => ({
-      params: { slug: node.slug },
-    })) || [];
-
+export const getStaticPaths: GetStaticPaths = async () => {
+  const paths = await categoryPaths();
   return {
     paths: paths,
     fallback: true,
   };
-}
+};
+
+export const getStaticProps = async (context: GetStaticPropsContext) => {
+  const categorySlug = context.params?.slug?.toString() as string;
```

Comment:
```suggestion
  const categorySlug = context.params?.slug?.toString()!;
```
Assert the type instead